cmd/helm/install.go

```diff
@@ -190,6 +191,8 @@ func newInstallCmd(c helm.Interface, out io.Writer) *cobra.Command {
 			}
 			inst.chartPath = cp
 			inst.client = ensureHelmClient(inst.client)
+			inst.wait = inst.wait || inst.atomic
```

Comment:
I don't understand why we need to set the wait flag when `--atomic` is set. From what I see here, there's nothing that the `--atomic` flag has a hard reliance on the wait flag being enabled.

Would you mind explaining the use case for this, and why `helm install/upgrade --atomic` requires the `--wait` flag to be set?

Reply:
Linking to the comment that answers this question in case anyone lands here with a question: https://github.com/helm/helm/pull/5395#issuecomment-477686053